src/common/common_service/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "risingwave_common_service"
```

Comment:
Would you explain why we need a new crate rather than the original service module?

Reply:
Circular dependency. And when I had to add the code to `common` before. Someone said `common` is very large. So I add a new crate